api/v1alpha1/helper.go

```diff
@@ -44,6 +44,15 @@ func GetOrdinalPodName(etcdObjMeta metav1.ObjectMeta, ordinal int) string {
 	return fmt.Sprintf("%s-%d", etcdObjMeta.Name, ordinal)
 }
 
+// GetAllPodNames returns the names of all pods for the Etcd.
+func GetAllPodNames(etcdObjMeta metav1.ObjectMeta, replicas int32) []string {
+	podNames := make([]string, 0, replicas)
+	for i := 0; i < int(replicas); i++ {
+		podNames = append(podNames, GetOrdinalPodName(etcdObjMeta, i))
+	}
+	return podNames
```

Comment:
The `for` loop in Go 1.22 now supports ranging directly over integers, as detailed here: [Go 1.22 Update](https://tip.golang.org/doc/go1.22#language). I suggest utilizing this feature for cleaner code:
```suggestion
	podNames := make([]string, replicas)
	for i := range int(replicas) {
		podNames[i] = GetOrdinalPodName(etcdObjMeta, i)
	}
	return podNames
```